### PR TITLE
RLM-1443 Reduce rpc-upgrades jobs to weekly runs

### PR DESCRIPTION
**Pull request by antonym**
Reduces rpc-upgrades jobs to weekly runs instead of
daily runs.

Issue: [RLM-1443](https://rpc-openstack.atlassian.net/browse/RLM-1443)

### CODE REVIEW

**Comment by antonym** [comment]
@mattt416 updated for your suggestion

**Comment by mattt416** [comment]
recheck_re_unit_tests

**Comment by mattt416** [comment]
recheck_re_unit_tests

**Comment by mattt416** [comment]
recheck_cit_unit_tests

File: rpc_jobs/rpc_upgrades.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@
     jira_project_key: "RLM"
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
+    CRON: "@weekly"
```
Comment (mattt416) [inline rpc_jobs/rpc_upgrades.yml]: Some of these are PR projects, so CRON will be ineffective on them.

**Pull request merged**